packages/auth-compat/test/integration/flows/custom.test.ts

```diff
@@ -137,6 +140,65 @@ describe('Integration test: custom auth', () => {
     expect(customUser!.uid).not.to.eql(anonUser!.uid);
   });
 
+  context('in passthrough mode', () => {
+    beforeEach(async () => {
+      const updatedConfig = await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'PASSTHROUGH'
+        })
+      );
+      expect(updatedConfig).to.eql({
+        signIn: { allowDuplicateEmails: false },
+        usageMode: 'PASSTHROUGH'
+      });
+    });
+
+    afterEach(async () => {
+      await updateEmulatorProjectConfig(
+        JSON.stringify({
+          usageMode: 'DEFAULT'
+        })
+      );
+    });
+
+    xit('signs in with custom token in passthrough mode', async () => {
+      const cred = await firebase.auth().signInWithCustomToken(customToken);
+      expect(firebase.auth().currentUser).to.eq(cred.user);
+      expect(cred.operationType).to.eq('signIn');
+
+      const { user } = cred;
+      expect(user!.isAnonymous).to.be.false;
+      expect(user!.uid).to.eq(uid);
+      expect((await user!.getIdTokenResult(false)).claims.customClaim).to.eq(
+        'some-claim'
+      );
+      expect(user!.providerId).to.eq('firebase');
+      const additionalUserInfo = cred.additionalUserInfo!;
+      expect(additionalUserInfo.providerId).to.be.null;
+      expect(additionalUserInfo.isNewUser).to.be.false;
+    });
+
+    xit('token can be refreshed in passthrough mode', async () => {
+      firebase.auth().setCustomTokenProvider({
+        async getCustomToken(): Promise<string> {
+          return JSON.stringify({
+            uid,
+            claims: {
+              customClaim: 'other-claim'
+            }
+          });
+        }
+      });
+      const { user } = await firebase.auth().signInWithCustomToken(customToken);
+      const origToken = await user!.getIdToken();
+      await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
Please add a comment explaining the need for the delay

Reply:
Ah, I think I copied this over from the non-passthrough refresh `'token can be refreshed'` test but it's not needed here. Removed in both auth/ and auth-compat/ tests!